patches/protonvpn-gui/application-id.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/protonvpn_gui/main.py b/protonvpn_gui/main.py
+index 96f1a11..312de6a 100644
+--- a/protonvpn_gui/main.py
++++ b/protonvpn_gui/main.py
+@@ -36,7 +36,7 @@ class ProtonVPNGUI(Gtk.Application):
+ 
+     def __init__(self):
+         super().__init__(
+-            application_id='com.protonvpn.www',
```

Comment:
Why this change? If the app id is com.protonvpn.www then so it is, this should be brought up upstreams instead. App ids in theory are not a user facing thing.

Note that changing the app id can unforeseen issues.

Reply:
this also probably require adding a `own-name` if that's needed as it wouldn't be granted by default.